sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/RequestValidator.java

```diff
@@ -94,6 +94,81 @@ public static void validateUpdateContent(Request contentRequestDto) {
               ERROR_CODE);
         }
       }
+      List<Map<String, Object>> assessmentData = (List<Map<String, Object>>) contentRequestDto.getRequest().get(JsonKey.ASSESSMENT_EVENTS);
+      System.out.println(assessmentData);
+      if (null != assessmentData && !assessmentData.isEmpty()) {
+        for (Map<String, Object> map : assessmentData) {
+          if (map.containsKey(JsonKey.COURSE_ID)) {
+            if (null == map.get(JsonKey.COURSE_ID)) {
+              throw new ProjectCommonException(
+                      ResponseCode.courseIdRequired.getErrorCode(),
+                      ResponseCode.courseIdRequiredError.getErrorMessage(),
+                      ERROR_CODE);
+            }
+          } else {
+            throw new ProjectCommonException(
+                    ResponseCode.courseIdRequired.getErrorCode(),
+                    ResponseCode.courseIdRequiredError.getErrorMessage(),
+                    ERROR_CODE);
+          }
+
+          if (map.containsKey(JsonKey.CONTENT_ID)) {
+            if (null == map.get(JsonKey.CONTENT_ID)) {
+              throw new ProjectCommonException(
+                      ResponseCode.contentIdRequired.getErrorCode(),
+                      ResponseCode.contentIdRequiredError.getErrorMessage(),
+                      ERROR_CODE);
+            }
+          } else {
+            throw new ProjectCommonException(
+                    ResponseCode.contentIdRequired.getErrorCode(),
+                    ResponseCode.contentIdRequiredError.getErrorMessage(),
+                    ERROR_CODE);
+          }
+
+          if (map.containsKey(JsonKey.BATCH_ID)) {
+            if (null == map.get(JsonKey.BATCH_ID)) {
+              throw new ProjectCommonException(
+                      ResponseCode.courseBatchIdRequired.getErrorCode(),
+                      ResponseCode.courseBatchIdRequired.getErrorMessage(),
+                      ERROR_CODE);
+            }
+          } else {
+            throw new ProjectCommonException(
+                    ResponseCode.courseBatchIdRequired.getErrorCode(),
+                    ResponseCode.courseBatchIdRequired.getErrorMessage(),
+                    ERROR_CODE);
+          }
+
+          if (map.containsKey(JsonKey.USER_ID)) {
+            if (null == map.get(JsonKey.USER_ID)) {
+              throw new ProjectCommonException(
+                      ResponseCode.userIdRequired.getErrorCode(),
+                      ResponseCode.userIdRequired.getErrorMessage(),
+                      ERROR_CODE);
+            }
+          } else {
+            throw new ProjectCommonException(
+                    ResponseCode.userIdRequired.getErrorCode(),
+                    ResponseCode.userIdRequired.getErrorMessage(),
+                    ERROR_CODE);
+          }
+
+          if (map.containsKey(JsonKey.ATTEMPT_ID)) {
+            if (null == map.get(JsonKey.ATTEMPT_ID)) {
```

Comment:
Use `StringUtils.isEmpty` - There is a chance of sending ATTEMPT_ID empty string.